awswrangler/_config.py

```diff
@@ -72,6 +82,7 @@ def __init__(self) -> None:
         self.verify = None
         for name in _CONFIG_ARGS:
             self._load_config(name=name)
+        self.distributed: bool = all(importlib.util.find_spec(pkg) for pkg in ("modin", "ray"))
```

Comment:
It feels awkward to add this to the config class given that it's not something we wish the user to actually configure but more something for us to distinguish between distributed and non-distributed wrangler code

Reply:
I would not be against giving them ability to turn it off... would be especially useful during development